### DISCUSSION TITLE
TP10 - Operaciones con Arreglos

### DISCUSSION THREAD

**Discussion by alengeor** [INGCOM-UNRN-P1/cursada-2023#215] [Preguntas y respuestas] [answered]
Buenas gente, como andan? Me surgo la duda, en el ejercicio de TP10 que pide analizar _las cuatro_ operaciones con arreglos, ¿cuales hicieron o piensan hacer? 
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/4059467e-624f-4a22-83de-0a780b8c162f)

**Comment by martinvilu** [comment, marked as answer]
En donde dice, “los cuatro” son los cuatro puntos.

Cargador de arreglos
Impresora de arreglos
Fusionadora de arreglos
Operaciones con arreglos
 *  Promedio
 *  Obtenga el valor mínimo
 *  Obtenga la posición del valor máximo
 *  Ordenar de menor a mayor

**Reply by alengeor** [reply]
Gracias Martín!
